swssconfig/sample/msn2700.32ports.buffers.json.j2

```diff
@@ -0,0 +1,423 @@
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_10G_40m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"18432",
+            "size":"36864",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_25G_40m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"21504",
+            "size":"39936",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_40G_40m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"23552",
+            "size":"41984",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_50G_40m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"23552",
+            "size":"41984",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_100G_40m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"35840",
+            "size":"54272",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_10G_300m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"30720",
+            "size":"49152",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_25G_300m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"53248",
+            "size":"71680",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_40G_300m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"75776",
+            "size":"94208",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_50G_300m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"75776",
+            "size":"94208",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
+    },
+    {
+        "BUFFER_PROFILE_TABLE:pg_lossless_100G_300m_profile": {
+            "pool":"[BUFFER_POOL_TABLE:ingress_lossless_pg_pool]",
+            "xon":"18432",
+            "xoff":"165888",
+            "size":"184320",
+            "dynamic_th":"1"
+        },
+        "OP": "SET"
```

Comment:
Please correct me if I understood it wrong:

size = xoff + xon implies that by allocating/freeing one memory allocation unit, we could jump from xon to xoff or vice versa.

Would this create lots of on/off activity when the traffic is marginal at xoff/xon threshold?

Would it be better to make size = xoff + xon + delta, so we have a range of delta to stay at either xon or xoff?

Thanks,
Ying